scripts/deploy.py

```diff
@@ -79,6 +81,20 @@
         print("We've been waiting for an hour. That seems bad. Failing now.")
         sys.exit(1)
 
-    print("Looks good to release! Pushing the tag now.")
+    print("Looks good to release!")
+
+    print("Now uploading to pypi.")
+
+    subprocess.check_output([
+        sys.executable, "setup.py", "sdist"
+    ])
+
+    subprocess.check_output([
+        sys.executable, "-m", "twine", "--config-file=./.pypirc",
+        "upload", "dist/*"
```

Comment:
Worth trashing the contents of the `dist` directory before you run the `sdist` step?

I can’t remember if it was Travis specifically, but I’ve had weirdness in the past when supposedly ephemeral containers got cached between runs.